cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -281,6 +281,14 @@
     mount options=(ro rbind) /snap/{,ubuntu-}core/*/var/lib/** -> /var/lib/**,
     umount /var/lib/snapd/,
     mount options=(move) /tmp/snapd.quirks_*/ -> /var/lib/snapd/,
+    # On classic systems with a setuid root snap-confine when run by non-root
```

Comment:
I like to group like rules together with different rule groups separated by spaces (I may not be 100% consistent with this, but it is what I like to do and did here). This rule is about silencing something related to quirks, so I put it there, without a space.